doc/deployment_examples/sinfonier/feeding_sinfonier.md

```diff
@@ -0,0 +1,408 @@
+* `cygnusagent.sinks.kafka-sink.broker_list` : Need the ip and port of your `Kafka` `Brokers`. See [next section](#section5.1) for more information about kafka.
+* `cygnusagent.sinks.kafka-sink.zookeeper_endpoint`: In this case, we are running `Zookeeper` in `localhost` with port 2181 (`Zookeeper` port). See [next section](#section5.2) for more information about zookeeper.
+* `cygnusagent.sinks.kafka-sink.data_model`: `Cygnus` parameter. Use dm-by-entity for a descriptive storage.
+
+Running properly all the structure (See [general procedure step-by-step](#section7) for do it properly) and updating some values in our `Entity` you can see how `Cygnus` persist the information.
+
+####<a name=”section4.2></a>Running
+Run `Cygnus` is possible with a command:
+```
+/path/to/flume/folder/bin/flume-ng agent --conf /path/to/flume/folder/conf -f /path/to/flume/folder/conf/kafka_agents/your_agent.conf -n cygnusagent -Dflume.root.logger=INFO,console
+```
+An own terminal is required for this service. Another way is using `nohup` at the start and `&`at the end for running the server in background (Use restricted when brokers are running properly and if you know how `nohup` works)
+```
+nohup /path/to/flume/folder/bin/flume-ng agent --conf /path/to/flume/folder/conf -f /path/to/flume/folder/conf/kafka_agents/your_agent.conf -n cygnusagent -Dflume.root.logger=INFO,console &
+```
+
+Once you have your `Cygnus` running and update some values on `Orion` the notification will be persisted. You will see logs like:
+```
+2016-XX-XX 09:13:57,365 (SinkRunner-PollingRunner-DefaultSinkProcessor) [INFO - com.telefonica.iot.cygnus.sinks.OrionKafkaSink.persistAggregation(OrionKafkaSink.java:264)] [kafka-sink] Persisting data at OrionKafkaSink. Topic (Book1_Book), Data ({"headers":[{"fiware-service":"def_serv"},{"fiware-servicePath":"def_servpath"},{"timestamp":1451981636718}],"body":{"attributes":[{"name":"title","type":"text","value":"Game of thrones: A song of ice and fire"},{"name":"pages","type":"integer","value":"985"},{"name":"price","type":"float","value":"23.50"}],"type":"Book","isPattern":"false","id":"Book1"}})
+2016-XX-XX 09:13:57,414 (SinkRunner-PollingRunner-DefaultSinkProcessor) [INFO - com.telefonica.iot.cygnus.sinks.OrionSink.process(OrionSink.java:196)] Finishing transaction (1451981632-289-0000000000)
+```
+As you can see, the information is persisted in a `topic` "Book1_Book" with `attributes` title, pages and price. The values match with the last update that we did in `Orion`.
+
+[Top](#top)
+
+##<a name="section5"></a>Kafka
+[Apache Kafka](http://kafka.apache.org/documentation.html#quickstart) is a distributed, partitioned, replicated commit log service. It provides the funcionality of a messaging system, but with a unique design.
+The use of `Kafka` for that purpose have two main pieces: `Zookeeper` and `brokers` (or servers, both names are correct). `Kafka` is needed for storage the translated information received from `Cygnus`.
```

Comment:
for storring the context information handled by the combination of Orion and Cygnus.